docs/meshstack.monitoring.md

```diff
@@ -180,3 +180,1664 @@ let example =
       : SonarScope
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Metrics Metadata
```

Comment:
d: Can we put this on its' own page, it's so long.

Reply:
Yeah, makes sense. Added a link from the monitoring page.